### PR TITLE
Add google analytics to pkgdown site.

### LINKED ISSUE #3432: Add google analytics to pkgdown site.
I'll do this, I'll just wait for a point at which you feel things are in an OK state to do so. So, just say the word! 🙂

### PR DESCRIPTION
**Pull request by batpigandme**
* Fixes #3432 
* Add Google Analytics to site.
* New pkgdown version means addition of `LICENSE.md` (and md friends) in `docs/LICENSE.html`
* n.b. Other re-rendering is just the addition of diacritical marks in the Author names.

### CODE REVIEW

File: docs/CONDUCT.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
+<!-- Generated by pkgdown: do not edit by hand -->
+<!DOCTYPE html>
+<html>
+  <head>
+  <meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+<title>/Users/maraaverick/dplyr/CONDUCT.md • dplyr</title>
```
Comment (jennybc) [inline docs/CONDUCT.html]: This path makes me think there's a problem in pkgdown? Search for "maraaverick" and you'll find a couple more.

Comment (batpigandme) [reply docs/CONDUCT.html]: _Damn_! Thanks, Jenny. I'm updating pkgdown again rn. I think it's fixed in the latest. And then, of course, I'll check for my name!

Comment (hadley) [reply docs/CONDUCT.html]: It's not fixed yet; I'd recommend just leaving it for now

Comment (batpigandme) [reply docs/CONDUCT.html]: Oh. I already changed it in 6de394b. Do you want me to put it back?

**Comment by krlmlr** [comment]
@batpigandme: Please merge when you think it's good.

**Pull request closed without merging**